src/tools/csg_stat_imc.cc

```diff
@@ -61,14 +64,12 @@ void Imc::Initialize() {
 
   // initialize non-bonded structures
   for (tools::Property *prop : _nonbonded) {
-    interaction_t *i = AddInteraction(prop);
-    i->_is_bonded = false;
+    interaction_t *i = AddInteraction(prop, false);
```

Comment:
```suggestion
bool bonded=false;
AddInteraction(prop, bonded);
```

Reply:
So we don't need to save the return value in i?

Reply:
That should fix the `unused variable 'i'` error as well.